src/components/NetworkSelector.tsx

```diff
@@ -0,0 +1,145 @@
+import { Menu, Transition } from "@headlessui/react"
```

Comment:
This file is mostly the same, I just moved it and updated it to use the new hooks.